pkg/scaling/scaledjob/scale_metrics.go

```diff
@@ -0,0 +1,186 @@
+				maxValueSum += metrics.maxValue
+				isActive = metrics.isActive
+				length++
+			}
+		}
+		if length != 0 {
+			queueLength = divideWithCeil(queueLengthSum, int64(length))
+			maxValue = divideWithCeil(maxValueSum, int64(length))
+		}
+	case "sum":
+		for _, metrics := range scalersMetrics {
+			if metrics.isActive {
+				queueLength += metrics.queueLength
+				maxValue += metrics.maxValue
+				isActive = metrics.isActive
+			}
+		}
+	default: // max
+		for _, metrics := range scalersMetrics {
+			if metrics.queueLength > queueLength && metrics.isActive {
+				queueLength = metrics.queueLength
+				maxValue = metrics.maxValue
+				isActive = metrics.isActive
+			}
+		}
+	}
+	maxValue = min(scaledJob.MaxReplicaCount(), maxValue)
+	logger.V(1).WithValues("ScaledJob", scaledJob.Name).Info("Checking if ScaleJob scalers are active", "isActive", isActive, "maxValue", maxValue, "MultipleScalersCalculation", scaledJob.Spec.ScalingStrategy.MultipleScalersCalculation)
+
+	return isActive, queueLength, maxValue
+}
+
+func getScalersMetrics(ctx context.Context, scalers []scalers.Scaler, scaledJob *kedav1alpha1.ScaledJob, logger logr.Logger, recorder record.EventRecorder) []scalerMetrics {
+	scalersMetrics := []scalerMetrics{}
+
+	// TODO refactor this, do chores, reduce the verbosity ie: V(1) and frequency of logs
+	// move relevant funcs getTargetAverageValue(), min() and divideWithCeil() out of scaler_handler.go
+	for _, scaler := range scalers {
+		var queueLength int64
+		var targetAverageValue int64
+		isActive := false
+		maxValue := int64(0)
+		scalerType := fmt.Sprintf("%T:", scaler)
+
+		scalerLogger := logger.WithValues("ScaledJob", scaledJob.Name, "Scaler", scalerType)
+
+		metricSpecs := scaler.GetMetricSpecForScaling()
+
+		// skip scaler that doesn't return any metric specs (usually External scaler with incorrect metadata)
+		// or skip cpu/memory resource scaler
+		if len(metricSpecs) < 1 || metricSpecs[0].External == nil {
+			continue
+		}
+
+		isTriggerActive, err := scaler.IsActive(ctx)
+		if err != nil {
+			scalerLogger.V(1).Info("Error getting scaler.IsActive, but continue", "Error", err)
+			recorder.Event(scaledJob, corev1.EventTypeWarning, eventreason.KEDAScalerFailed, err.Error())
+			scaler.Close()
+			continue
+		}
+
+		targetAverageValue = getTargetAverageValue(metricSpecs)
+
+		metrics, err := scaler.GetMetrics(ctx, "queueLength", nil)
+		if err != nil {
+			scalerLogger.V(1).Info("Error getting scaler metrics, but continue", "Error", err)
+			recorder.Event(scaledJob, corev1.EventTypeWarning, eventreason.KEDAScalerFailed, err.Error())
+			scaler.Close()
+			continue
+		}
+
+		var metricValue int64
+
+		for _, m := range metrics {
+			if m.MetricName == "queueLength" {
+				metricValue, _ = m.Value.AsInt64()
+				queueLength += metricValue
+			}
+		}
+		scalerLogger.Info("Scaler Metric value", "isTriggerActive", isTriggerActive, "queueLength", queueLength, "targetAverageValue", targetAverageValue)
```

Comment:
```suggestion
		scalerLogger.V(1).Info("Scaler Metric value", "isTriggerActive", isTriggerActive, "queueLength", queueLength, "targetAverageValue", targetAverageValue)
```
I think that the Debug output is better for this kind of message